capabilities.json

```diff
@@ -530,5 +541,15 @@
             }]
         }
     },
-    "supportsHighlight": true
+    "tooltips": {
+        "supportedTypes": {
+            "default": true,
+            "canvas": true
+        },
+        "roles": ["tooltips"]
+    },
+    "supportsHighlight": true,
```

Comment:
We have to remove `supportsHighlight` support for now, since it will break current users expectations. And there is no a mechanism for now to set default interaction behavior.

Reply:
It's been added 8 years ago. I'm unable to remove it as it will break user expectations.